third_party/com_github_aws_checksums/aws_checksums.BUILD.bazel

```diff
@@ -0,0 +1,42 @@
+# Description:
+#   AWS CheckSums
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+cc_library(
+    name = "aws-checksums",
+    srcs = glob([
+        "include/aws/checksums/*.h",
+        "include/aws/checksums/private/*.h",
+        "source/*.c",
+    ]) + [
+        "crc_hw.c",
+    ],
+    hdrs = [],
```

Comment:
I would expect a header file. Is that not the case?

Reply:
No headers were specified in the original: https://github.com/tensorflow/io/blob/4d70341df05ddce09c257d4ecaaaba1873594105/third_party/aws-checksums.BUILD#L10-L25